packages/bitcore-client/bin/wallet-send

```diff
@@ -50,11 +50,39 @@ const main = async () => {
       from: lastAddress,
       token
     };
-    const tx = await wallet.newTx(params);
+    let tx;
+    const txTemp = await wallet.newTx(params);
```

Comment:
So if I'm reading this right, you aren't mutating this tx, why bother with the txTemp variable declaration rather than just give it a new value with line 78. Then you can drop the variable `txTemp` and the else statement altogether.